transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpUtils.java

```diff
@@ -412,6 +412,23 @@ public static URI getRequestURI(URI requestURI, String hostHeader, IoSession ses
         return uri;
     }
 
+    /*
+     * Returns whether there is Connection: close header
+     *
+     * @param list of Connection header values
+     * @return true if Connection: close header is part of the values
+     */
+    public static boolean hasCloseHeader(List<String> connectionValues) {
+        if (connectionValues != null) {
+            for (String value : connectionValues) {
```

Comment:
I think we can start using stream api
connectionValues.stream().anyMatch(v -> v.equalsIgnoreCase("close"));